### DISCUSSION TITLE
Reference error THREE is undefined

### DISCUSSION THREAD

**Discussion by sadrajafari** [scenerystack/community#27] [Q&A]
Hey, I hope you are doing well. I am wondering why I am seeing this error for my new project when I want to use THREE. It would be great if you help me to solve this problem with THREE.js

Thanks

**Comment by jessegreenberg** [comment]
Hi @sadrajafari - three.js is a separate library with its own setup. I would recommend checking out the official three.js documentation for creating a scene (https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene).
